### ISSUE TITLE
HTM integration

### ISSUE THREAD

**Issue by RemDelaporteMathurin** [IdahoLabResearch/HyPAT#9]
Hi HyPAT team!

I think some of the hydrogen transport properties in HyPAT could be corrected by integrating it with [HTM](https://github.com/RemDelaporteMathurin/h-transport-materials).

HTM is an open source database of H properties (@FuerstT and @natethegreatINL have been contributing).
It can be used to [plot Arrhenius properties with matplotlib](https://h-transport-materials.readthedocs.io/en/latest/user/examples.html#accessing-the-internal-database-basic) by filtering the database.

I think it would help refactoring HyPAT and would reduce the line count slightly.

Let me know what you guys think

I'm happy to come up with a PR if that's of interest

**Comment by FuerstT** [comment]
Great idea @RemDelaporteMathurin! 

If you can please start a PR and work with @natethegreatINL on integration, that would be an excellent upgrade to the current overview plots tab.

**Comment by natethegreatINL** [comment]
@RemDelaporteMathurin, I would be happy to help with this integration, is there a place you would like me to get started?

**Comment by RemDelaporteMathurin** [comment]
> @RemDelaporteMathurin, I would be happy to help with this integration, is there a place you would like me to get started?

I'll have a look in the code to see where this would come into play

**Comment by RemDelaporteMathurin** [comment]
Right. I managed to fiddle a bit with the source code. Not very familliar with `tkinter`...

I've managed to have a proof of concept of how to integrate HyPAT with HTM.

Here's an example for just tungsten, aluminium and Inconel 600:

<img width="705" alt="image" src="https://github.com/IdahoLabResearch/HyPAT/assets/40028739/9863fe73-7e52-4c17-afcf-78f650152774">

Note: the graphs are totally decoupled from the left column here.

I can push the code for this prototype to a branch if you guys are interested.

On a side note: I would avoid using xlsx files for several reasons:

- They can only be opened with the right software (Excel, Libreoffice..)
- They cannot be visualised in version control (because they are binary files)

Since all of the xlsx files are simple 2 columns data, I would instead suggest using .csv files that can be opened with any text reader (and Excel!) and are much easier to work with from a programmatic point of view (can be opened with `pandas` or `numpy` in one line of code).

Happy to discuss this further.

**Comment by RemDelaporteMathurin** [comment]
EDIT: I added a bunch of materials.
Now the database sometimes contains several entries for one material.
In my proof of concept, I just chose the first entry for the material.

I'll leave you guys choose what to do:
- Option 1: choose from an exact reference
- Option 2: take the mean of the properties in the database for this material

<img width="709" alt="image" src="https://github.com/IdahoLabResearch/HyPAT/assets/40028739/3ee7f367-a1a8-4d31-ad67-9cf3d2e8ba04">
